pkg/util/log.go

```diff
@@ -127,7 +129,7 @@ func WithUserID(userID string, l log.Logger) log.Logger {
 // its details.
 func WithTraceID(traceID string, l log.Logger) log.Logger {
 	// See note in WithContext.
-	return log.With(l, "trace_id", traceID)
+	return log.With(l, "traceID", traceID)
```

Comment:
> I'm also renaming span_id to spanID because we have inconsistency with weaveworks server which uses spanID.

Note to myself: I guess you meant `traceID` and not `spanID`.